### PR TITLE
AusOceanTV: Add Developer README

### PR DESCRIPTION
**Pull request by ao-david**
This readme should help developers with some of the project specific technologies used in the development of AusOcean TV.

Describes #350

### CODE REVIEW

File: cmd/ausoceantv/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+TailwindElements are defined in individual directories to group an element
+with its style sheet, these directories should be stored in the src parent
+directory. To create the element, create a new typescript file, and a new css
+file. Use the following template to get started:
+``` TSX
+import {html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {TailwindElement} from '../shared/tailwind.element';
+
+import style from './test.component.scss?inline'; // # See NOTE
```
Comment (scott97) [inline cmd/ausoceantv/README.md:49]: From my testing this should be
```js
import style from './shared/tailwind.global.css?inline';
```

**Review by scott97 — CHANGES_REQUESTED** [review]
One small change, otherwise it is good to go :+1:

**Review by scott97 — APPROVED** [review]

File: cmd/ausoceantv/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# AusOcean TV
+AusOcean TV is a subscription service for accessing AusOceans premium streams.
```
Comment (trekhopton) [inline cmd/ausoceantv/README.md:2]: *AusOcean's

**Comment by trekhopton** [comment]
Cool that you can preview markdown in github using the view file option.

File: cmd/ausoceantv/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+$ npm run dev
+```
+To learn more about Vite see: [Vite](https://vite.dev).
+
+### Lit Elements
+Lit elements are a thin wrapper on web-components and provide a lightweight
+way to define reactive and reusable components. AusOcean uses Typescript to
+in our Lit Elements to improve type safety.
+
+To Allow for Tailwind styling to work with Lit components, AusOcean uses a
```
Comment (trekhopton) [inline cmd/ausoceantv/README.md:35]: Allow -> allow

**Review by trekhopton — CHANGES_REQUESTED** [review]
Nice work. This will be helpful. Just a few typos.

**Comment by ao-david** [comment]
> Nice work. This will be helpful. Just a few typos.

Done: :relaxed:

**Review by trekhopton — APPROVED** [review]

**Pull request merged**